src/gui/transferlistsortmodel.h

```diff
@@ -55,12 +55,14 @@ class TransferListSortModel final : public QSortFilterProxyModel
 
 private:
     int compare(const QModelIndex &left, const QModelIndex &right) const;
+    void saveSubSortColumn();
+    void loadSubSortColumn();
 
     bool lessThan(const QModelIndex &left, const QModelIndex &right) const override;
     bool filterAcceptsRow(int sourceRow, const QModelIndex &sourceParent) const override;
     bool matchFilter(int sourceRow, const QModelIndex &sourceParent) const;
 
     TorrentFilter m_filter;
     mutable int m_subSortColumn = -1;
-    mutable int m_sortColumn = -1;
+    mutable int m_lastSortColumn = -1;
```

Comment:
Why did you change it? Isn't it "current" sort column?

Reply:
no, it will be only updated on the first call of `less than` after sort changes, which is never for the empty model.